### DISCUSSION TITLE
Occasionally,  I can't access vercel.com and my website www.slaysi.com after deployment.

### DISCUSSION THREAD

**Discussion by nusdavid** [vercel/community#4050] [Help] [answered]
### Summary

I'm David, I'm developer of www.slaysi.com.
Occasionally, I can't access vercel.com and my website www.slaysi.com after deployment. It's weird because this issue has occurred recently, this didn't occur before. 
Could you check DNS system from your side? Thank you.

### Steps to Reproduce

**ISSUE 1**
+ Deploy new release to www.slaysi.com
+ F5 www.slaysi.com several times on browser, sometime it responds, sometime it won't respond randomly
+ Note that www.slaysi.com is integrated with Cloudflare. However, I access subdomains that Vercel assigned to my website, I got the same issue:

slaysi-fe-slaysi.vercel.app
slaysi-fe-git-production-slaysi.vercel.app

**ISSUE 2**
Occasionally, I can't access vercel.com .

**Comment by nusdavid** [comment]
![Screenshot from 2023-09-13 09-54-11](https://github.com/vercel/community/assets/7609985/f2d3900a-7f9d-42ab-b5b7-b12383b791c4)
![Screenshot from 2023-09-13 15-17-28](https://github.com/vercel/community/assets/7609985/a720a368-a7c6-4ac9-8703-4ce5b3d828a1)

**Comment by amyegan** [comment]
Hey @nusdavid. I wasn't able to reproduce the issue. Site loading problems could be caused lots of things, such as network issues, firewall settings, and some browser extensions. Can you check if this problem happens in other browsers or on another network, if possible?

**Reply by nusdavid** [reply]
Hi @amyegan , 
Thanks for your response, let me try.

**Reply by nusdavid** [reply, marked as answer]
Hi @amyegan,
The root cause may from my side, Vietnamese ISP Viettel
https://twitter.com/vercel_support/status/1588384330753400832

Thanks again.
